src/app/containers/Image/index.jsx

```diff
@@ -77,6 +77,7 @@ const ImageContainer = ({ blocks }) => {
         ratio={ratio}
         src={rawImageSrc}
         width={width}
+        lazyLoad
```

Comment:
I note that you've opened up this issue: https://github.com/bbc/simorgh/issues/1765 
Are UX & Product happy about this work happening in a separate PR - so that in the interim between this PR being merged and the ticket being completed, all images on live articles will be lazyloaded?